### ISSUE TITLE
When i keep MPAndroid PieChart in FrameLayout after swipe half chart going out of screen

### ISSUE THREAD

**Issue by eswaraaa** [Ramotion/paper-onboarding-android#39]
In PaperOnboardingEngine.java implemented PieChart to show in each screen swipe. Once i press right swipe PieChart going bottom of screen and only half chart is showing. Please help to fix this issue.

protected ViewGroup createContentTextView(PaperOnboardingPage PaperOnboardingPage) {
        LayoutInflater vi = LayoutInflater.from(mAppContext);
        ViewGroup contentTextView = (ViewGroup) vi.inflate(R.layout.onboarding_text_content_layout, mContentTextContainer, false);
...
pieChart = (PieChart) contentTextView.getChildAt(1);
        pieChart.setRotationEnabled(false);
        pieChart.setUsePercentValues(true);
..
..
}

**Comment by golovin47** [comment]
Hello, @eswaraaa. To resolve this issue you may download the examples from this repository and 
put your PieChart there. The reason of this issue is probably in your implementation.

**Comment by eswaraaa** [comment]
Hi golovin47, 
Thanks for your quick response. Issue is occurring only when i keep PieChart in screen.   I implemented piechart in this xml layout layout.onboarding_text_content_layout

and referring it here:
PaperOnboardingEngine.java
protected ViewGroup createContentTextView(PaperOnboardingPage PaperOnboardingPage) {
LayoutInflater vi = LayoutInflater.from(mAppContext);
ViewGroup contentTextView = (ViewGroup) vi.inflate(R.layout.onboarding_text_content_layout, mContentTextContainer, false);
...
pieChart = (PieChart) contentTextView.getChildAt(1);
pieChart.setRotationEnabled(false);
pieChart.setUsePercentValues(true);
..
..
}

should i need to include any extra code in any place or in ViewGroup 4 animate content text createContentCenteringVerticalAnimation method 
 // 4 animate content text
        ViewGroup newContentText = createContentTextView(newElement);
        mContentTextContainer.addView(newContentText);
        AnimatorSet contentTextShowAnimation = createContentTextShowAnimation(
                mContentTextContainer.getChildAt(mContentTextContainer.getChildCount() - 2), newContentText);

**Comment by eswaraaa** [comment]
> Hello, @eswaraaa. To resolve this issue you may download the examples from this repository and
> put your PieChart there. The reason of this issue is probably in your implementation.

Hi golovin47,
Thanks for your quick response. I already integrated your code to my project. Issue is occurring only when i keep PieChart in screen. I implemented piechart in this xml layout layout.onboarding_text_content_layout

and referring it here:
PaperOnboardingEngine.java
protected ViewGroup createContentTextView(PaperOnboardingPage PaperOnboardingPage) {
LayoutInflater vi = LayoutInflater.from(mAppContext);
ViewGroup contentTextView = (ViewGroup) vi.inflate(R.layout.onboarding_text_content_layout, mContentTextContainer, false);
...
pieChart = (PieChart) contentTextView.getChildAt(1);
pieChart.setRotationEnabled(false);
pieChart.setUsePercentValues(true);
..
..
}

should i need to include any extra code in any place or in ViewGroup 4 animate content text createContentCenteringVerticalAnimation method
// 4 animate content text
ViewGroup newContentText = createContentTextView(newElement);
mContentTextContainer.addView(newContentText);
AnimatorSet contentTextShowAnimation = createContentTextShowAnimation(
mContentTextContainer.getChildAt(mContentTextContainer.getChildCount() - 2), newContentText);
